src/WebJobs.Script/Workers/Http/HttpWorkerDescription.cs

```diff
@@ -31,12 +34,14 @@ public override void ApplyDefaultsAndValidate(string workerDirectory, ILogger lo
             if (!string.IsNullOrEmpty(DefaultWorkerPath) && !Path.IsPathRooted(DefaultWorkerPath))
             {
                 DefaultWorkerPath = Path.Combine(WorkerDirectory, DefaultWorkerPath);
+                ThrowIfFileNotExists(DefaultWorkerPath, nameof(DefaultWorkerPath));
```

Comment:
@anthonychu - This will now throw explicit file not found exception if file specified in `DefaultWorkerPath` or `DefaultExecutablePath` does not exist